src/components/basket/BasketEmptyMessage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function BasketEmpty(props) {
+    return (
+        <div id="basket__empty"
+            className={"basket__empty " + (Object.keys(props.orders).length === 0 ? '' : 'disabled')}>
```

Comment:
Condition could be written as slightly shorter as `!Object.keys(props.orders).length`